src/app/(client)/onboarding/components/cta-card.tsx

```
@@ -0,0 +1,44 @@
+import { useUserStore } from '@/store/user-store';
+import SITE_MAP from '@/constants/site-map.constant';
+import { FormEvent } from 'react';
+
+const CtaCard = ({ setCookie }: { setCookie: () => void }) => {
+  const user = useUserStore((state) => state.user);
+  const router = useRouter();
+
+  const handleClickCtaButton = async (e: FormEvent<HTMLFormElement>): Promise<void> => {
+    e.preventDefault();
+    setCookie();
+    if (!!user.id) {
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)